dashboard/static/styles/main.styl

```diff
@@ -13,7 +15,7 @@ main {
 
 section {
     margin-top: 20px;
-    margin-bottom: 20px;
+    margin-bottom: @margin-top;
```

Comment:
Ah, yeah. We could do that too. I called out `margin-top` and `margin-bottom` separately just in case there's another section somewhere that needs to set different left and right margins. We wouldn't want to override those.